consensus/istanbul/backend/backend.go

```diff
@@ -1013,3 +1030,20 @@ func (sb *Backend) UpdateReplicaState(seq *big.Int) {
 		sb.replicaState.NewChainHead(seq)
 	}
 }
+
+// recordBlockProductionTimes records information about the block production cycle and reports it through the CSVRecorder
+func (sb *Backend) recordBlockProductionTimes(blockNumber, txCount, gasUsed, round uint64) {
+	cycle := time.Since(sb.cycleStart)
+	sb.cycleStart = time.Now()
+	sleepGauge := sb.sleepGauge
+	consensusGauge := metrics.Get("consensus/istanbul/core/consensus_commit").(metrics.Gauge)
+	verifyGauge := metrics.Get("consensus/istanbul/core/verify").(metrics.Gauge)
+	blockConstructGuage := metrics.Get("miner/worker/block_construct").(metrics.Gauge)
```

Comment:
`guage` here too.  There are others as well, so to fix them all go through and search for `guage`.

Reply:
🤦 I'll go fix them.